### DISCUSSION TITLE
alsa_output: Decoder is too slow; playing silence to avoid xrun

### DISCUSSION THREAD

**Discussion by goweropolis** [MusicPlayerDaemon/MPD#1387] [Q&A]
Hi there,

I am receiving the error message "alsa_output: Decoder is too slow; playing silence to avoid xrun" in the log.

This is occurring when listening to streaming radio stations, not when listening to music on a network drive.  The same error is occurring with multiple radio stations so I don't believe it's the radio stations themselves.

I am running MPD version 023.4 on a Raspberry Pi 4 which is connecting to my wifi network via a 2.4ghz connection.

I am attaching the relevant files to assist.
[mpd.conf.txt](https://github.com/MusicPlayerDaemon/MPD/files/7766076/mpd.conf.txt)
[mpd-version.txt](https://github.com/MusicPlayerDaemon/MPD/files/7766077/mpd-version.txt)
[log.txt](https://github.com/MusicPlayerDaemon/MPD/files/7766078/log.txt)

This bug here [https://github.com/MusicPlayerDaemon/MPD/issues/1263](url) is the same problem I'm having, which is why I tried to adjust my mpd.conf file to include the buffer_time and period_time settings to try and correct, but it didn't seem to fix the problem.

Any suggestions or assistance would be greatly appreciated.

Thanks,
Andy

**Comment by MaxKellermann** [comment]
I can't reproduce this because your stream URL gives me "403 Forbidden".

**Reply by goweropolis** [reply]
Here's a couple of different stations that I am using where the error occurs.
This is cut and pasted from a m3u playlist that is in my MPD database.
You can see for QMFM that I was trying different links to the stream to see if they worked better.

#EXTINF:1, 103.5 QMFM
https://18803.live.streamtheworld.com/CHQMFMAAC_SC
#https://18803.live.streamtheworld.com/CHQMFMAAC.aac
#https://19253.live.streamtheworld.com/CHQMFMAAC_SC
#https://playerservices.streamtheworld.com/api/livestream-redirect/CHQMFMAAC_SC
#EXTINF:1, 104.3 The Breeze
https://newcap.leanstream.co/CHLGFM
